config/samples/core_v1alpha2_manualscalertrait.yaml

```diff
@@ -3,9 +3,9 @@ kind: ManualScalerTrait
 metadata:
   name: manualscalertrait-sample
 spec:
-  replicaCount: 5
+  replicaCount: 15
```

Comment:
An example with 15 replica is too large.
Make it 1?

Reply:
This is to test the mutation webhook

Reply:
make it 1~3 for test